addon/Feeds/MetadataFeed.js

```diff
@@ -18,9 +18,16 @@ module.exports = class MetadataFeed extends Feed {
    * the state
    */
   getInitialMetadata(reason) {
-    return PlacesProvider.links.getRecentlyVisited().then(links => {
+    // Get initial topsites metadata
+    return PlacesProvider.links.getTopFrecentSites().then(links => {
```

Comment:
might be worth renaming that `getRecentlyVisited` query, it's really easy to confuse it. maybe something like `getPossibleHighlights` or something?